openbb_platform/extensions/economy/openbb_economy/markets/markets_router.py

```diff
@@ -0,0 +1,53 @@
+"""Economy Markets Router."""
+
+from openbb_core.app.model.command_context import CommandContext
+from openbb_core.app.model.obbject import OBBject
+from openbb_core.app.provider_interface import (
+    ExtraParams,
+    ProviderChoices,
+    StandardParams,
+)
+from openbb_core.app.query import Query
+from openbb_core.app.router import Router
+
+router = Router(prefix="/markets")
+
+# pylint: disable=unused-argument
+
+
+# @router.command(model="MarketCategoryQuotes")
+# async def quote_category(
+#     cc: CommandContext,
+#     provider_choices: ProviderChoices,
+#     standard_params: StandardParams,
+#     extra_params: ExtraParams,
+# ) -> OBBject:
+#     """Get quotes for a specific category of markets."""
+#     return await OBBject.from_query(Query(**locals()))
+
+
+# @router.command(model="MarketSymbolQuotes")
+# async def quote_symbol(
+#     cc: CommandContext,
+#     provider_choices: ProviderChoices,
+#     standard_params: StandardParams,
+#     extra_params: ExtraParams,
+# ) -> OBBject:
+#     """Get quotes for a specific symbol."""
+#     return await OBBject.from_query(Query(**locals()))
+
+
+@router.command(model="MarketHistorical")
+async def historical(
+    cc: CommandContext,
+    provider_choices: ProviderChoices,
+    standard_params: StandardParams,
+    extra_params: ExtraParams,
+) -> OBBject:
+    """
+    Get historical price data for a symbol within a market category.
+
+    The market categories available are exchange rates, stock market indexes,
```

Comment:
Should this functionality be split up into the existing router functions, like `index.price.historical`, `crypto.price.historical`, `currency.price.historical`?

Reply:
I agree with darren.  This api is wacky, but we can get snapshots without a ticker (f'https://api.tradingeconomics.com/markets/index?c={api_key}' or for the specific symbol:

f'https://api.tradingeconomics.com/markets/historical/aapl:us,gac:com?c={api_key}'

So this should go into snapshot and historical for crypto/index/stock. This also probably warrants a new bond/government/snapshot  for  f'https://api.tradingeconomics.com/markets/bond?c={api_key}'

Reply:
That's a very fair point. I don't know tbh, although it covers those assets, it's not limited to them - and TE doesn't seem to offer any options for filtering.
Thoughts @jmaslek @minhhoang1023 ?

Reply:
The page was outdated when I 1st commented! I agree with that approach @jmaslek. Will restructure accordingly.
So, iiuc: 
   - TE Historical will serve crypto/index/equity historical.
   - TE Snapshot will serve economy/index snapshot. --> should we create a snapshot endpoint under economy for the bonds?

The expectation is that the user can then use (eg) `index.historical` to get historical data for any given bond?